tests/integration/kubernetes/run_kubernetes_tests.sh

```diff
@@ -161,12 +167,12 @@ if [ -f "${arch_config_file}" ]; then
 	mapfile -d " " -t K8S_TEST_UNION <<< "${arch_k8s_test_union}"
 fi
 
-if policy_tests_enabled; then
-	ensure_yq
-	run_policy_specific_tests
-	add_policy_to_successful_tests
-else
+# if policy_tests_enabled; then
+# 	ensure_yq
+# 	run_policy_specific_tests
+# 	add_policy_to_successful_tests
+# else
 	info "Policy tests are disabled on this platform"
-fi
+# fi
```

Comment:
Please remove all commented out code (I spot some more below). If necessary, feel free to paste it into a GitHub issue and reference that issue URL here.